src/Microsoft.Health.Fhir.R4.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -14,16 +14,16 @@ namespace Microsoft.Health.Fhir.Api.Features.Resources.Bundle
     /// </summary>
     public partial class BundleHandler
     {
-        private Dictionary<Hl7.Fhir.Model.Bundle.HTTPVerb, List<RouteContext>> GenerateRequestDictionary()
+        private Dictionary<Hl7.Fhir.Model.Bundle.HTTPVerb, Dictionary<RouteContext, int>> GenerateRequestDictionary()
```

Comment:
We should just use something like a list of tuples here instead of dictionaries. For example: `List<(RouteContext routeContext, int order)>`

Reply:
👍